### PR TITLE
Add in default sorting to export & import

### PR DESCRIPTION
**Pull request by lamtu1**
Overriding the original layout to add in default sorting for both the `importer` & `exporter` for Bulkrax.

### CODE REVIEW

**Comment by lamtu1** [comment]
fixes #2941

**Comment by lamtu1** [comment]
`DEMO:` Showing the sorting default in bulkrax.

https://github.com/OregonDigital/OD2/assets/111306182/c4116777-23d9-442b-ad92-427be62a4672

File: app/views/bulkrax/exporters/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          </tbody>
+        </table>
+      </div>
+      <% else %>
+        <p>No exporters have been created.</p>
+      <% end %>
+  </div>
+</div>
+
+<script>
```
Comment (straleyb) [inline app/views/bulkrax/exporters/index.html.erb:59]: Can this script be moved over into a js file? That would be more close to what we have normalized in the application.

File: app/views/bulkrax/importers/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        </table>
+      </div>
+      <% else %>
+        <p>No importers have been created.</p>
+      <% end %>
+  </div>
+</div>
+
+<script>
+  $(function() {
```
Comment (straleyb) [inline app/views/bulkrax/importers/index.html.erb:67]: Same here

File: app/views/bulkrax/exporters/index.html.erb (hunk 1 above)
Comment (lamtu1) [reply app/views/bulkrax/exporters/index.html.erb:59]: Will do! since both of the file are using the same order, having one script js file should be good.

**Review by straleyb — APPROVED** [review]

**Pull request merged**